.github/workflows/build.yml

```
@@ -79,6 +79,7 @@ jobs:
           - { go: "1.22.4", goos: "windows", goarch: "386" }
           - { go: "1.22.4", goos: "windows", goarch: "amd64" }
           - { go: "1.22.4", goos: "solaris", goarch: "amd64" }
+          - { go: "1.22.4", goos: "darwin", goarch: "amd64" }
```

I know this wasn't here before but we might as well add it since M1 macbook support is quite common now. It goes to show you how little milage this repo gets.
```suggestion
          - { go: "1.22.4", goos: "darwin", goarch: "amd64" }
          - { go: "1.22.4", goos: "darwin", goarch: "arm64" }
```